apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -203,7 +205,7 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
       <RequiresConfirmationController
         eventType={eventType}
         seatsEnabled={seatsEnabled}
-        metadata={eventType.metadata}
+        metadata={formMethods.getValues("metadata")}
```

Comment:
There are some cases when we need the original `eventType` prop alongside `formMethods` values.
For ex. having both the count of webhooks and other form values in the same component.

what we can do is instead of calling `formMethods.getValues("some_property_name")` everywhere, we just do `formValues = formMethods.getValues()` saving the execution calls and cleaning up things. 
but formMethods.getValues isn't reactive (value change doesn't cause re-render), so for some cases we will still be using formMethods.watch().

Reply:
I have tested a bit with using the `formValues` strategy, as expected it is not reliable. We should use `formMethods.getValues("some_prop")`